app/Filament/Forms/Components/Licenses.php

```diff
@@ -96,6 +98,10 @@ private function generateBlockForLicenseType(LicenseType $licenseType): Fieldset
                         }
 
                         $notification->send();
+
+                        if ($licenseType == LicenseType::ConversationalAi && Cache::has('ai-users-count') && Feature::active('ai_utilization')) {
+                            Cache::forget('ai-users-count');
+                        }
```

Comment:
We should probably do this within the `grantLicense` and `revokeLicense` used methods above

Reply:
done